tests/unit/conftest.py

```diff
@@ -1,40 +0,0 @@
-import pytest
-import sqlalchemy
-
-from report.db import SESSION
-
-
-@pytest.fixture
-def db_session(db_engine):
```

Comment:
We don't use this one at all, if we need it again we can just copied it over again.